src/components/Avatar/Avatar.tsx

```diff
@@ -168,7 +168,7 @@ class Avatar extends UIComponent<Extendable<IAvatarProps>, any> {
           <Label
             styles={{ root: styles.avatarNameContainer }}
             as="div"
-            content={getInitials(name)}
+            content={getInitials && getInitials(name || '')}
```

Comment:
Not sure if we need the check here as the getInitials prop has default value.

Reply:
that's a good point, let me addess it..